docs/authorized_keys.md

```diff
@@ -0,0 +1,33 @@
+---
+id: authorized_keys
+title: Authorized_Keys Configuration
+---
+# Signatory's Authorized Key Authentication Feature
+
+Signatory's remote policy service feature allows custom policy schemes beyond simple request and operation lookup to be implemented externally. This feature is authenticated using a signature, which requires the public key to be added to the authorized_keys list.
+
+## Authentication
+
+Authentication is the process of verifying the identity of a user or system. In the context of cryptographic keys, authentication ensures that the keys are legitimate and belong to the authorized parties.
```

Comment:
maybe instead of "In the context of cryptographic keys, authentication ensures that the keys are legitimate and belong to the authorized parties." something like "Client applications can provide signed requests to a service proving their identity through possession of the private key."

Reply:
changed as requested